pkg/hub/submarineragent/controller.go

```diff
@@ -687,12 +689,44 @@ func (c *submarinerAgentController) createGNConfigMapIfNecessary(ctx context.Con
 		logger.Infof("Globalnet is disabled in the managedClusterSet namespace %q", brokerNamespace)
 	}
 
-	if err := globalnet.CreateConfigMap(ctx, c.controllerClient, brokerObj.Spec.GlobalnetEnabled,
-		brokerObj.Spec.GlobalnetCIDRRange, brokerObj.Spec.DefaultGlobalnetClusterSize, brokerNamespace); err != nil {
-		return errors.Wrapf(err, "error creating globalnet configmap on Broker")
+	configMap, err := globalnet.NewGlobalnetConfigMap(brokerObj.Spec.GlobalnetEnabled,
+		brokerObj.Spec.GlobalnetCIDRRange, brokerObj.Spec.DefaultGlobalnetClusterSize, brokerNamespace)
+	if err == nil {
+		configMap.Labels[BackupLabelKey] = BackupLabelValue
+		err = c.controllerClient.Create(ctx, configMap)
 	}
 
-	return nil
+	return errors.Wrapf(err, "error creating globalnet configmap on Broker")
+}
+
+func (c *submarinerAgentController) updateBackupLabelOnGnConfigMap(ctx context.Context, namespace string) error {
+	return errors.Wrapf(retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		logger.Infof("Adding backup label to globalnet configmap in %q", namespace)
```

Comment:
I think this will cause too much logging, especially since this is only for upgrade. The logging on L724 should suffice.

Reply:
++. 

Also, how frequently this API will be called?

Reply:
Aha, sorry, this was for debugging/testing and missed to remove it before pushing.